src/index.js

```diff
@@ -1,36 +1,20 @@
-const SDK_VERSION = "0.0.12";
+export const SDK_VERSION = "0.0.12";
```

Comment:
Why did we have to export it?

Reply:
This was left over from when I was trying some stuff out. Thanks for catching that!